packages/icons/src/svg/dimension-category-option-groupset-16.svg

```diff
@@ -0,0 +1 @@
+<svg height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m14 1c.5522847 0 1 .44771525 1 1v12c0 .5522847-.4477153 1-1 1h-12c-.55228475 0-1-.4477153-1-1v-12c0-.55228475.44771525-1 1-1zm0 1h-12v12h12zm-5 8-1 1-1-1zm2-2-1 1h-4l-1-1zm2-2-1 1h-8l-1-1z" fill="#404b5a" fill-rule="evenodd"/></svg>
```

Comment:
`fill` rule not correctly set to `#010101` in the export process. Apologies!

Reply:
Thanks for the heads up, I'll update it 👍.

Reply:
Fixed in ab026e1